source/release-notes.txt

```diff
@@ -11,6 +11,46 @@ Release Notes
    :class: twocols
 
 
+1.10.0 Changelog
+---------------
+
+*Released October 1, 2024*
```

Comment:
Adding a note here to keep top of mind that the release date is still a little bit TBD.